Keas.Mvc/Controllers/Api/KeySerialsController.cs

```diff
@@ -298,7 +307,6 @@ public async Task<IActionResult> Revoke(int id)
             // clear out assignment
             var assignment = keySerial.KeySerialAssignment;
             _context.KeySerialAssignments.Remove(assignment);
-            keySerial.KeySerialAssignmentId = null;
```

Comment:
@sprucely This isn't need anymore because the remove null's it out?

Reply:
correct